mathesar_ui/src/components/AppHeader.svelte

```diff
@@ -76,7 +77,7 @@
         <DropdownMenu
           triggerAppearance="ghost"
           size="small"
-          closeOnInnerClick={true}
+          closeOnInnerClick={false}
```

Comment:
This change is so the user can interact with the theme switcher. It is only noticed when choosing the documentation links, as internal link clicks already close the menu by means of page navigation.